imagetest/test_suites/storage_perf/iops_read_test.go

```diff
@@ -0,0 +1,56 @@
+//go:build cit
+// +build cit
+
+package storageperf
+
+import (
+	"encoding/json"
+	"math"
+	"os/exec"
+	"strconv"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest/utils"
+)
+
+// TestReadIOPS checks that read IOPS are around the value listed in public docs.
+func TestReadIOPS(t *testing.T) {
+	symlinkRealPath := ""
+	diskPartition, err := getMountDiskPartition(HyperdiskSize)
+	if err == nil {
+		symlinkRealPath = "/dev/" + diskPartition
+	} else {
+		errorString := err.Error()
+		symlinkRealPath, err = getMountDiskPartitionSymlink()
+		if err != nil {
+			errorString += err.Error()
+			t.Fatalf("failed to find symlink to mount disk with any method: errors %s", errorString)
+		}
+	}
+
+	if !utils.CheckLinuxCmdExists("fio") {
+		if err := installFio(); err != nil {
+			t.Fatal(err)
+		}
+	}
+
+	// Arbitrary file read size, less than the size of hte hyperdisk in GB.
+	fileReadSizeString := strconv.Itoa(HyperdiskSize/10) + "G"
+	iopsJson, err := exec.Command("fio", "--name=read_iops_test", "--filename="+symlinkRealPath, "--filesize="+fileReadSizeString, "--time_based", "--ramp_time=2s", "--runtime=1m", "--ioengine=libaio", "--direct=1", "--verify=0", "--randrepeat=0", "--bs=4k", "--iodepth=256", "--rw=randread", "--iodepth_batch_submit=256", "--iodepth_batch_complete_max=256", "--output-format=json").Output()
+	if err != nil {
+		t.Fatalf("fio command failed with %v", err)
```

Comment:
Being a little nitpicky here, but the formatting could be adjusted a bit to match how we usually pass errors up the chain. Maybe something like:

"fio command failed: %v"

Or maybe (verbose):
"fio command failed with error: %v"